src/interactions/login.ts

```diff
@@ -22,6 +26,6 @@ function login({
   if (isValid) {
     localStorage.setItem('TOKEN', jwtToken);
     localStorage.setItem('REFRESH_TOKEN', refreshToken);
-    window.location.href = url;
+    history.push(url);
```

Comment:
One suggestion could you change history to `window.history` and remove the interface?

Reply:
never mind. I see this is a global instance of react router